src/components/App/App.js

```diff
@@ -7,20 +7,45 @@ import SingleMovie from "../SingleMovie/SingleMovie";
 import HomeButton from "../HomeButton/HomeButton";
 
 function App() {
-  const mockData = movieData.movies;
-  const singleMockData = singleMovieData.movies;
-  const [movies, setMovies] = useState(mockData);
+  const [movies, setMovies] = useState([]);
   const [singleMovie, setSingleMovie] = useState(null);
```

Comment:
I would encourage you to use the data type you are expecting in state instead of null - promoting consistent data types is important! Tools like TypeScript would not allow you to do this.